--- configure	2017-04-13 03:55:54.000000000 +0200
+++ configure_patched_no_autodetected_externals	2017-07-09 02:35:54.126646400 +0200
@@ -1485,18 +1485,6 @@
     transcoding_example
 "
 EXTERNAL_AUTODETECT_LIBRARY_LIST="
-    bzlib
-    iconv
-    libxcb
-    libxcb_shm
-    libxcb_shape
-    libxcb_xfixes
-    lzma
-    schannel
-    sdl
-    sdl2
-    securetransport
-    xlib
     zlib
 "
 
@@ -1592,17 +1580,6 @@
     videotoolbox
 "
 HWACCEL_AUTODETECT_LIBRARY_LIST="
-    audiotoolbox
-    cuda
-    cuvid
-    d3d11va
-    dxva2
-    nvenc
-    vaapi
-    vda
-    vdpau
-    videotoolbox_hwaccel
-    xvmc
 "
 
 HWACCEL_LIBRARY_NONFREE_LIST="
